.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
- - [ ] closes #xxxx
- - [ ] tests added / passed
- - [ ] passes ``git diff upstream/master -u -- "*.py" | flake8 --diff``
- - [ ] whatsnew entry
+- [ ] closes #xxxx
+- [ ] tests added / passed
+- [ ] passes `git diff upstream/master -- "*.py" | flake8 --diff`
```

Comment:
I checked the `git diff` [documentation](https://git-scm.com/docs/git-diff), which appears to be language agnostic, going back to git [version 1.8.5](https://git-scm.com/docs/git-diff/1.8.5). I didn't see anything that would cause issues (like `--patch` not being default).

However, I only tested the changes on Linux, not macOS or Windows. @gfyoung I'm happy to revert. I'm also happy to leave this open and seek verification from mac and windows users. Which do you prefer?